server/main.lua

```diff
@@ -1888,6 +1937,25 @@ function sendToDiscord(color, name, message, footer)
 	end
 end
 
+function sendIncidentToDiscord(color, name, message, footer)
+	if ClockinWebhook == '' then

```

Comment:
IncidentWebhook

Reply:
@xSisyphusx

Reply:
My mistake will fix this right now

Reply:
Wait - this is just the clock in webhook for the time clock - this should be correct.

Reply:
> Wait - this is just the clock in webhook for the time clock - this should be correct.

No, you're using the function sendIncidentToDiscord and sending it to ClockinWebhook, it needs to be sent to IncidentWebhook

Reply:
@MonkeyWhisper - I have corrected this issue. My fault, for whatever reason I was having a brain fart.